docs/user_guide/email_notifications/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Email Notifications
+sidebar: sidebars/documentation.html
+---
+
+# Email Notifications
+
+## Types of Email Notifications
+
+In addition to the Email Notifications that Foreman provides:
+
+* Puppet run summary (Daily/Weekly/Monthly)
+* Puppet errors
+
+Katello provides a few addition reports:
+
+* Katello Host Advisory (Daily/Weekly/Monthly) - A report of all of the Errata applicable to all readable Content Hosts
+* Katello Promote Errata - A report generated at Content View promotion time showing what new Errata have been promoted and how many Content Hosts are applicable.
```

Comment:
Small clarification, the mail shows unapplied errata from hosts registered to that CVE, not only new.  I originally thought it might be too hard to calculate, but I could change it later to use the same date-based approach as syncing.